integration-tests/search-apprenticeship/Search by Posted In Last N Days.postman_collection.json

```diff
@@ -0,0 +1,256 @@
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "fd0ce585-4aa2-4b18-8d0e-0b1086b0d3ac",
+						"exec": [
+							"pm.test(\"Has response code of bad request\", function () {",
+							"    pm.response.to.have.status(400);",
+							"});",
+							"",
+							"pm.test(\"Has got exact number of errors\", function () {",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.requestErrors.length === 1);",
+							"});",
+							"",
+							"pm.test(\"Has correct error message\", function () {",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.requestErrors[0].errorMessage).to.eql(\"'Posted In Last Number Of Days' must be greater than or equal to '0'.\");",
+							"});",
+							"",
+							"pm.test(\"Has correct error code\", function () {",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.requestErrors[0].errorCode).to.eql(\"30105\");",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "Ocp-Apim-Subscription-Key",
+						"value": "{{apim-key}}",
+						"type": "text"
+					}
+				],
+				"url": {
+					"raw": "{{apim-gateway-url}}/{{live-vacancies-path}}/v1/apprenticeships/search?postedInLastNumberOfDays=-1",
+					"host": [
+						"{{apim-gateway-url}}"
+					],
+					"path": [
+						"{{live-vacancies-path}}",
+						"v1",
+						"apprenticeships",
+						"search"
+					],
+					"query": [
+						{
+							"key": "postedInLastNumberOfDays",
+							"value": "-1"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Valid Posted in Last Number of Days",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "70c56274-ffa6-4bf8-90e1-5a7b3789acb9",
+						"exec": [
+							"pm.test(\"Has response code of good request\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"Responses posted in last number of days\", function() {",
+							"    var jsonData = pm.response.json();",
+							"    ",
+							"    var totalReturned = jsonData.totalReturned;",
+							"    var currentDate = new Date();",
+							"    currentDate.setHours(0,0,0,0);",
+							"    ",
+							"    tests[\"This is the currentDate \" + currentDate] = currentDate > 0;",
+							"    tests[\"totalReturned \" + totalReturned] = totalReturned > 0;",
+							"    ",
+							"    for(var i = 0; i < totalReturned; i++)",
+							"    {",
+							"        var postedDataField = jsonData.results[i].postedDate;",
+							"        var postedDate = new Date(postedDataField);",
+							"        postedDate.setHours(0,0,0,0);",
+							"        tests[\"This is the postedDate \" + postedDate] = postedDate > 0;",
+							"        ",
+							"        // Add 3 days",
+							"        postedDate.setDate(postedDate.getDate() + 3);",
```

Comment:
Why are we adding 3 days to postedDate?

Reply:
It was to match the filter which was set at 3, 0 is matches on today's date only, the plus 3 just ensures all matches are within the extra 3 days.  I could change the filter to 0 and remove the +3 to set it at the smallest possible matches if that makes more sense?